src/dao/createDatabase.js

```diff
@@ -0,0 +1,26 @@
+require('require-self-ref')
```

Comment:
This whole file is gross but exists to limitations with knex/migrations. You can't create databases within a migration because it fails on the initial connect attempting to connect to a database that doesn't exist. Instead, we run this script just before the migration to ensure the database exists for the migrations.

Reply:
Yeah that sucks. Maybe we should open an issue on knex to see if we can't get it supported.

Reply:
Might be good to move this into another folder, possibly along the lines of `src/dao/_migration-helpers`. Just to make it obvious that it's a utility that shouldn't be imported.